src/index.html

```diff
@@ -1,13 +1,284 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bose</title>
+    <link 
+      rel="shortcut icon" 
+      href="images/Favicon.png" 
+      type="image/x-icon" 
+    >
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container">
+        <div class="header__content">
+          <div class="header__top">
+            <a class="header__logo logo link" href="./index.html">
```

Comment:
```suggestion
            <a class="header__logo logo link" href="#">
```
bc it yours reloads a page